cmd/main.go

```
@@ -1 +1,34 @@
-package cmd
+package main
+
+import (
+	"fmt"
+	"log"
+	"todo-clean/database"
+	"todo-clean/delivery"
+	"todo-clean/repository"
+	"todo-clean/usecase"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+
+	dbConn, err := database.ConnectDB()
+	//
+	if err != nil {
+		log.Panicln("failed to connect database", err)
```

Why did you use `log panic` instead of `log fatal`